### PR TITLE
SPCTR-2797: Improvement: Removal of extra varialble for props

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

 Removal of extra varialble for props by replacing with spread operator.

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/buttons-child/edit.js [hunk 1]
```diff
@@ -77,13 +77,13 @@ const ButtonsChildComponent = ( props ) => {
 			<DynamicFontLoader { ...{ attributes } } />
 			{ isSelected && (
 				<Settings
-					parentProps={ props }
+					{ ...props }
 					state={ state }
 					setStateValue={ setStateValue }
 					deviceType={ deviceType }
```
Comment (rahulvarma722) [inline src/blocks/buttons-child/edit.js:83]: We can get DeviceType from props so remove deviceType

File: src/blocks/call-to-action/settings.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ import ResponsiveSlider from '@Components/responsive-slider';
 import { useDeviceType } from '@Controls/getPreviewType';
 
 const Settings = ( props ) => {
-	props = props.parentProps;
+
 	const { setAttributes, attributes } = props;
 	const deviceType = useDeviceType();
```
Comment (rahulvarma722) [inline src/blocks/call-to-action/settings.js:26]:
```suggestion
	const { setAttributes, attributes, deviceType } = props;
```
and also remove importing

File: src/blocks/cf7-designer/edit.js [hunk 3]
```diff
@@ -291,8 +291,8 @@ const UAGBCF7 = ( props ) => {
 		<>
 			<DynamicCSSLoader { ...{ blockStyling } } />
 			<DynamicFontLoader { ...{ attributes } } />
-			{ isSelected && <Settings parentProps={ props } deviceType={ deviceType } /> }
-			<Render parentProps={ props } />
+			{ isSelected && <Settings { ...props } deviceType={ deviceType } /> }
```
Comment (rahulvarma722) [inline src/blocks/cf7-designer/edit.js:294]:
```suggestion
			{ isSelected && <Settings { ...props } /> }
```
remove deviceType

File: src/blocks/column/edit.js [hunk 4]
```diff
@@ -107,8 +107,8 @@ const ColumnComponent = ( props ) => {
 	return (
 		<>
 			<DynamicCSSLoader { ...{ blockStyling } } />
-			{ isSelected && <Settings parentProps={ props } deviceType={ deviceType } /> }
-			<Render parentProps={ props } />
+			{ isSelected && <Settings { ...props } deviceType={ deviceType } /> }
```
Comment (rahulvarma722) [inline src/blocks/column/edit.js:110]: Remove deviceType

File: src/blocks/columns/edit.js [hunk 5]
```diff
@@ -221,8 +221,8 @@ const ColumnsComponent = ( props ) => {
 	return (
 		<>
 			<DynamicCSSLoader { ...{ blockStyling } } />
-			{ isSelected && <Settings parentProps={ props } deviceType={ deviceType } /> }
-			<Render parentProps={ props } />
+			{ isSelected && <Settings { ...props } deviceType={ deviceType } /> }
```
Comment (rahulvarma722) [inline src/blocks/columns/edit.js:224]: Remove deviceType

File: src/blocks/faq-child/render.js [hunk 6]
```diff
@@ -14,7 +14,6 @@ const Render = ( props ) => {
 	}, [] );
 
 	const state = props.state;
-	props = props.parentProps;
 
 	const { attributes, setAttributes } = props;
```
Comment (rahulvarma722) [inline src/blocks/faq-child/render.js:18]: you can also write like this and test this also
	const { attributes, setAttributes, state } = props;

File: src/blocks/faq/edit.js [hunk 7]
```diff
@@ -250,8 +250,8 @@ const FaqComponent = ( props ) => {
 		<>
 			<DynamicFontLoader { ...{ attributes } } />
 			<DynamicCSSLoader { ...{ blockStyling } } />
-			{ isSelected && <Settings parentProps={ props } deviceType={ deviceType } /> }
-			<Render parentProps={ props } />
+			{ isSelected && <Settings { ...props } deviceType={ deviceType } /> }
```
Comment (rahulvarma722) [inline src/blocks/faq/edit.js:253]: Remove devicetype

File: src/blocks/buttons-child/edit.js (hunk 1 above)
Comment (rajmohanreddy774) [reply src/blocks/buttons-child/edit.js]: Removed

File: src/blocks/call-to-action/settings.js (hunk 2 above)
Comment (rajmohanreddy774) [reply src/blocks/call-to-action/settings.js]: Removed

File: src/blocks/cf7-designer/edit.js (hunk 3 above)
Comment (rajmohanreddy774) [reply src/blocks/cf7-designer/edit.js]: Removed

File: src/blocks/column/edit.js (hunk 4 above)
Comment (rajmohanreddy774) [reply src/blocks/column/edit.js]: Removed

File: src/blocks/faq/edit.js (hunk 7 above)
Comment (rajmohanreddy774) [reply src/blocks/faq/edit.js]: Removed

File: src/blocks/faq-child/render.js (hunk 6 above)
Comment (rajmohanreddy774) [reply src/blocks/faq-child/render.js]: Done

**Review by rahulvarma722 — APPROVED** [review]

File: src/blocks/countdown/render.js [hunk 8]
```diff
@@ -14,12 +14,8 @@ const Render = ( props ) => {
 		};
 	}, [] );
 
-	const countdownRef = props.countdownRef;
-
-	props = props.parentProps;
-
 	const {
-		attributes: { block_id, showLabels, labelDays, labelHours, labelMinutes, labelSeconds, timerEndAction }, deviceType
+		attributes: { block_id, name, showLabels, labelDays, labelHours, labelMinutes, labelSeconds, timerEndAction }, deviceType, countdownRef
```
Comment (rahulvarma722) [inline src/blocks/countdown/render.js:18]: Keep this as it is 	const countdownRef = props.countdownRef;
because ref not working when you destruct

Comment (rajmohanreddy774) [reply src/blocks/countdown/render.js]: done

**Review by rahulvarma722 — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**